File: solutions/javascript/resistor-color-duo/1/resistor-color-duo.js
//
// This is only a SKELETON file for the 'Resistor Color Duo' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export function decodedValue([a, b, ..._]) {
    return ResistorValues.indexOf(a) * 10 + ResistorValues.indexOf(b);
}

const ResistorValues = [
    "black",
    "brown",
    "red",
    "orange",
    "yellow",
    "green",
    "blue",
    "violet",
    "grey",
    "white",
];

File: solutions/python/resistor-color-duo/1/resistor_color_duo.py

COLORS =  [ "black",
            "brown",
            "red",
            "orange",
            "yellow",
            "green",
            "blue",
            "violet",
            "grey",
            "white", ]


def value(colors):
    return int(str(COLORS.index(colors[0])) + str(COLORS.index(colors[1])))
